Documentation/batch-scripts/mentoring.py

```
@@ -0,0 +1,436 @@
+                ON uor.role_id = r.id
+               AND r.organization_id = o.id
+            WHERE uor.updated_at > %s
+        """, (last_run,))
+
+    rows = cur.fetchall()
+    logging.info(f"Fetched {len(rows)} org roles")
+
+    users = defaultdict(lambda: {"organizations": defaultdict(list)})
+
+    for row in rows:
+        user_id, org_id, org_name, org_code, role_id, role_name, tenant_code, updated_at = row
```

Ruff (0.13.1)

Unpacked variable `updated_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)